tests/conftest.py

```diff
@@ -22,62 +22,92 @@
 
 import os
 
-from ansys.dpf.core import (
-    AvailableServerContexts,
-    LicenseContextManager,
-    connect_to_server,
-    load_library,
-)
+from ansys.dpf.core import upload_file_in_tmp_folder
 import pytest
 
-CONTAINER_SERVER_PORT = 6780
-STR_DPF_SOUND = "dpf_sound.dll"
-STR_DPF_SOUND_DLL = "dpf_sound.dll"
+from ansys.sound.core.server_helpers import connect_to_or_start_server
 
 
 def pytest_configure():
-    pytest.data_path_flute_in_container = "C:\\data\\flute.wav"
-    pytest.data_path_flute2_in_container = "C:\\data\\flute2.wav"
-    pytest.data_path_flute_nonUnitaryCalib_in_container = "C:\\data\\flute_nonUnitaryCalib.wav"
-    pytest.data_path_sharp_noise_in_container = "C:\\data\\sharp_noise.wav"
-    pytest.data_path_sharper_noise_in_container = "C:\\data\\sharper_noise.wav"
-    pytest.data_path_rough_noise_in_container = "C:\\data\\rough_noise.wav"
-    pytest.data_path_rough_tone_in_container = "C:\\data\\rough_tone.wav"
-    pytest.data_path_fluctuating_noise_in_container = "C:\\data\\fluctuating_noise.wav"
-    pytest.data_path_fluctuating_tone_in_container = "C:\\data\\fluctuating_tone.wav"
-    pytest.data_path_white_noise_in_container = "C:\\data\\white_noise.wav"
-    pytest.data_path_accel_with_rpm_in_container = "C:\\data\\accel_with_rpm.wav"
-    pytest.data_path_flute_psd_in_container = "C:\\data\\flute_psd.txt"
-    pytest.data_path_swl_project_file_in_container = (
-        "C:\\data\\SoundPowerLevelProject_hemisphere_2025R1_20243008.spw"
-    )
-    pytest.data_path_swl_project_file_with_calibration_in_container = (
-        "C:\\data\\SoundPowerLevelProject_hemisphere_signalsWithCalibration_2025R1_20240919.spw"
-    )
-    pytest.data_path_sound_composer_spectrum_source_in_container = (
-        "C:\\data\\AnsysSound_Spectrum_v3_-_nominal_-_dBSPLperHz_2024R2_20241121.txt"
-    )
-    pytest.data_path_flute_nonUnitaryCalib_as_txt_in_container = (
-        "C:\\data\\flute_nonUnitaryCalib_as_text_2024R2_20241125.txt"
-    )
-    pytest.data_path_rpm_profile_as_wav_in_container = "C:\\data\\RPM_profile_2024R2_20241126.wav"
-    pytest.data_path_rpm_profile_as_txt_in_container = "C:\\data\\RPM_profile_2024R2_20241126.txt"
 
+    # We're using a PyAnsys Sound here to connect to the server based on whether
+    # we're in a docker or local configuration
+    # There are tests for the function connect_to_or_start_server
+    # that are independent from the configuration,
+    # that's why we authorize the use of this function here
+    server = connect_to_or_start_server(use_license_context=True)
 
-@pytest.fixture(scope="session")
-def dpf_sound_test_server():
-    port_in_env = os.environ.get("ANSRV_DPF_SOUND_PORT")
-    if port_in_env is not None:
-        port = int(port_in_env)
-    else:
-        port = CONTAINER_SERVER_PORT
+    # # Get the current directory of the conftest.py file
+    base_dir = os.path.dirname(__file__)
```

Comment:
@a-bouth How about doing ``base_dir = os.path.join(os.path.dirname(__file__), "data")`` instead, and then also ``pytest.temporary_folder = base_dir`` at the end? (If I understand correctly). That would avoid copying "data" X times below (in case we ever change that folder name for example, it will be faster to update)

Reply:
@ansaminard good idea, it's modified